dev/io.openliberty.cdi.4.0.internal_fat/fat/src/io/openliberty/cdi40/internal/fat/bce/beancreatorlookup/permissions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<permissions xmlns="https://jakarta.ee/xml/ns/jakartaee"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="https://jakarta.ee/xml/ns/jakartaee
+        https://jakarta.ee/xml/ns/jakartaee/permissions_9.xsd"
+    version="9">
+    <!-- Update to version=10 when implemented -->
+    <!-- Work around WELD-2705 -->
+    <permission>
+        <class-name>java.lang.RuntimePermission</class-name>
```

Comment:
Has the new weld release fixed this?

Reply:
It's fixed in some places but not others. In some cases the CDI would give access to reflection APIs which should be protected by J2S, so in those cases we still need a permissions.xml. I'll check whether this one is actually needed.